sources/freshdesk/__init__.py

```diff
@@ -0,0 +1,67 @@
+from typing import Iterable, List, Optional
+
+import dlt
+from dlt.common.typing import TDataItem
+from dlt.sources import DltResource
+
+from .helpers import create_auth_headers, get_endpoint, paginated_response
+
+
+@dlt.source
+def freshdesk_source(
+    endpoints: Optional[List[str]] = None,
```

Comment:
could you put all endpoints ["agents", "companies", "contacts", "groups", "roles", "time_entries"] to settings.py file and set them here as defaults.

see other sources, it's our standard for now.

Reply:
done